JobRunner/CatalogCache.py

```diff
@@ -2,14 +2,19 @@
 
 
 class CatalogCache(object):
-    def __init__(self, config):
-        self.admin_token = config.admin_token
-        self.catalog_url = config.catalog_url
-        self.catalog = Catalog(self.catalog_url, token=self.admin_token)
+
```

Comment:
I checked https://github.com/kbase/execution_engine2/blob/develop/lib/execution_engine2/utils/catalog_cache.py so we are passing in the catalog as a dependency there already!